account/user.go

```diff
@@ -0,0 +1,153 @@
+package account
+
+import (
+	"time"
+
+	"github.com/almighty/almighty-core/models"
+	"github.com/goadesign/goa"
+	"github.com/jinzhu/gorm"
+	uuid "github.com/satori/go.uuid"
+	"golang.org/x/net/context"
+)
+
+// User describes a User(single email) in any system
+type User struct {
+	models.Lifecycle
+	ID         uuid.UUID `sql:"type:uuid default uuid_generate_v4()" gorm:"primary_key"` // This is the ID PK field
+	Email      string    `sql:"unique_index"`                                            // This is the unique email field
```

Comment:
Thanks, But what should we do in delete API itself ? it will do soft delete, right ? I have made few changes [here](https://github.com/aslakknutsen/almighty-core/pull/4) for this topic